tools/ecs/composer.json

```diff
@@ -1,8 +1,6 @@
 {
     "require": {
-        "symplify/easy-coding-standard": "^10.3",
-        "slevomat/coding-standard": "^7.2",
-        "squizlabs/php_codesniffer": "^3.7"
+        "brick/coding-standards": "dev-main"
```

Comment:
Should this still be in `tools/ecs`? How will we handle this if we include Psalm in the coding-standards repo?

Reply:
Yes it should stay isolated because `brick/coding-standards` is actually a composer package too that just wrap ECS and some additional packages for specific sniffers.

However you raise a good point about psalm. This meta package cannot group  multiple tools into one package as it would fall into the dependency problem we tried to avoid in the first place.

The best option IMO would be to let each brick libraries require the tools independently and only use the coding-standards repo to share tools configuration files and github workflows.

This would bring some flexibility. 

Downside is, every time there's a new tool version, each brick libraries have to be updated one by one. Could be automated at some point though. 

AFAIU this what doctrine is doing.